src/all/deviantart/src/eu/kanade/tachiyomi/extension/all/deviantart/DeviantArt.kt

```diff
@@ -176,7 +191,35 @@ class DeviantArt : HttpSource() {
         return Jsoup.parse(body.string(), request.url.toString(), Parser.xmlParser())
     }
 
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        val artistInTitlePref = ListPreference(screen.context).apply {
+            key = ArtistInTitle.PREF_KEY
+            title = "Artist name in manga title"
+            entries = ArtistInTitle.values().map { it.text }.toTypedArray()
+            entryValues = ArtistInTitle.values().map { it.name }.toTypedArray()
+            summary = "%s"
```

Comment:
clarify in summary to clear database in advanced settings after changing this pref, and it will only apply to things not in currently library.